src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -380,7 +600,7 @@ space, there exists an isometry from the orthogonal complement of a nonzero sing
 def linear_isometry_equiv.from_orthogonal_span_singleton
   (n : ℕ) [fact (finrank 𝕜 E = n + 1)] {v : E} (hv : v ≠ 0) :
   (𝕜 ∙ v)ᗮ ≃ₗᵢ[𝕜] (euclidean_space 𝕜 (fin n)) :=
```

Comment:
Can you rewrite this definition so that the type produced is `orthonormal_basis (fin n) 𝕜 (𝕜 ∙ v)ᗮ`?  (Should just be a matter of dropping the `repr`.)